xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -176,6 +178,7 @@ void CPVRClients::Unload(void)
   m_strPlayingClientName = "";
 
   m_clientMap.clear();
+  m_loadedEvent.Reset();
```

Comment:
does this signal the waiting thread? haven't used this high level event implementation much